app/scripts/modules/datepicker.js

```diff
@@ -81,6 +81,8 @@ angular.module('strap.position', [])
 
 angular.module('modified.datepicker', ['strap.position'])
 
+var today = new Date;
+var minDate = new Date(today.getFullYear(), 0, 1, 0, 0, 0, 0)
```

Comment:
Adding `var` in between the module is breaking the datepicker since the next statement starts with `.`  Please don't use var put it directly in the object